cumulus/polkadot-parachain/polkadot-parachain-lib/src/cli.rs

```diff
@@ -111,6 +141,48 @@ impl Cli {
 	}
 }
 
+impl SubstrateCli for Cli {
+	fn impl_name() -> String {
+		Self::executable_name()
+	}
+
+	fn impl_version() -> String {
+		env!("SUBSTRATE_CLI_IMPL_VERSION").into()
+	}
+
+	fn description() -> String {
+		format!(
+			"The command-line arguments provided first will be passed to the parachain node, \n\
+			and the arguments provided after -- will be passed to the relay chain node. \n\
+			\n\
+			Example: \n\
+			\n\
+			{} [parachain-args] -- [relay-chain-args]",
+			Self::executable_name()
+		)
+	}
+
+	fn author() -> String {
+		env!("CARGO_PKG_AUTHORS").into()
+	}
+
+	fn support_url() -> String {
+		"https://github.com/paritytech/polkadot-sdk/issues/new".into()
+	}
+
+	fn copyright_start_year() -> i32 {
+		2017
+	}
```

Comment:
need a way to override those values

Reply:
Added a `CliConfig` for this. Also renamed `CommandConfig` to `RunConfig`